projects/addon-table/components/reorder/reorder.template.html

```diff
@@ -15,7 +15,15 @@
                     class="t-icon"
                     [src]="options.icons.drag"
                 />
-                {{ item }}
+                <ng-container
+                    *polymorpheusOutlet="
+                        container ? container : primitiveValue as template;
```

Comment:
No need to fallback, you can just add default content as a function that stringifies $implicit: `({$implicit}) => String($implicit)`